src/decaylanguage/dec/dec.py

```diff
@@ -131,8 +131,8 @@ def __init__(self, *filenames):
             # Content of input file(s)
             self._dec_file = stream.read()
         else:
-            self._dec_file_names = None
-            self._dec_file = None
+            self._dec_file_names = None  # type: ignore
```

Comment:
```suggestion
            self._dec_file_names = []
```

Reply:
This one is easy, so let me help you trivially.

Reply:
It will want to know what's inside the list, so quieting it with `type: list`. Ideally it should be `type: List[str]` I expect, but don't want to bother importing things from typing just yet.

Reply:
Ahh, you were right, this wasn't the definition. 🤦